PyInstaller/depend/bindepend.py

```diff
@@ -875,6 +876,36 @@ def get_python_library_path():
     # Applies only to non Windows platforms.
 
     if is_unix:
+
+        # Search if it is an Anaconda distribution
+        if re.findall('/anaconda', sys.executable):
+
+            # Relative anaconda path to the lib
+            rel_lib_path = 'lib/libpython3.7m.so.1.0'
+
+            # Split the path of the python executable
+            # to determine what the full anaconda name is
+            # and what paths preceed it.
+            splits = sys.executable.split('/anaconda')
```

Comment:
Again this will not work on e.g. Windows.

Reply:
This has not been used